pylti/tests/test_flask.py

```diff
@@ -173,9 +182,11 @@ def test_access_to_oauth_resource(self):
         self.assertFalse(self.has_exception())
 
     def test_access_to_oauth_resource_name_passed(self):
+        # pylint: disable=invalid-name
         """
         Check that name is returned if passed via initial request
         """
+        # pylint: disable=maybe-no-member
```

Comment:
I don't know what this message is, but you should try to fix the cause instead of just disabling the message.  Also, you have disables at the same scope in two different places